lib/index.jsx

```diff
@@ -66,7 +76,7 @@ class DatetimeRangePicker extends Component {
     return {
       ...baseProps,
       ...inputProps,
-      defaultValue: this.props.startDate,
+      value: this.state.start,
```

Comment:
May want to potentially support something like `defaultStartDate` & `defaultEndDate`

Reply:
That's a good idea. Please see: https://github.com/snamoah/react-datetime-range-picker/pull/7/commits/59b6c18c9057ff293469dde27364490b5e3014a1